Clean up slow code

vincetiu8 opened the pull request:
This cleans up the slow code from the Goblet cell to integrate it better with the dev environment.

vincetiu8 committed:
Clean up slow code

This cleans up the slow code from the Goblet cell to integrate it better with the dev environment.

Graphcalibur reviewed (changes requested):
Couple of issues:
- In the level 3 map, enemies that go through the arteries/veins no longer have their sprite made invisible (at least, this occurred for the red artery).
- "Save Settings" and "Done" can't be accessed in the tower preview menu anymore
- Enemies that are slowed seem to be slowed indefinitely instead of going back to normal after the specified slow duration.

vincetiu8 committed:
Fix bugs

vincetiu8 committed:
Add a scroll position

This adds a 'scroll position' attribute that can be used to scroll up and down the attributes. The attributes are automatically cut off when they disappear from the page.
